src/background.js

```diff
@@ -1,24 +1,20 @@
-let namespace = window.browser || chrome
+const runtime = chrome || window.browser
+
+//register handler for communication with content script
+runtime.runtime.onMessage.addListener((request, sender, sendResponse) => {
+
+  const runtime = chrome || window.browser
```

Comment:
why was that defined twice?

Reply:
behaviour of service workers, the outer const is removed over time. drove me crazy....